master/internal/core.go

```diff
@@ -187,6 +187,16 @@ func (m *Master) getMasterLogs(c echo.Context) (interface{}, error) {
 	return entries, nil
 }
 
+// @Summary Get an aggregated view of resource allocation during the given time period (CSV).
+// @Tags Cluster
+// @ID get-raw-resource-allocation-csv
+// @Accept  json
+// @Produce  text/csv
+// @Param   start query string true "Start time to get allocations for"
+// @Param   end query string true "End time to get allocations for"
+// @Success 200 {} string "experiment_id,kind,username,labels,slots,start_time,end_time,seconds"
```

Comment:
This could probably bear to be a bit more verbose ("A CSV file containing the fields...").

Reply:
```suggestion
// @Success 200 {} string "A CSV file containing the fields experiment_id,kind,username,labels,slots,start_time,end_time,seconds"
```